src/ShowTheShortcut.csproj

```diff
@@ -27,6 +27,9 @@
     <RootNamespace>ShowTheShortcut</RootNamespace>
     <AssemblyName>ShowTheShortcut</AssemblyName>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <StartAction>Program</StartAction>
+    <StartProgram>$(DevEnvDir)\devenv.exe</StartProgram>
+    <StartArguments>/rootsuffix Exp</StartArguments>
```

Comment:
Unrelated change to make it unnecessary to manually setup debugging every time you clone. I added as I was testing.